src/app/components/FrostedGlassPromo/withData.tsx

```diff
@@ -18,8 +18,20 @@ import styles from './styles';
 import { FormattedPromo, ImageProps, PromoProps } from './types';
 import { OptimoBlock } from '../../models/types/optimo';
 
-const buildImageProperties = (image?: ImageProps) => {
-  if (!image) return null;
+const buildImageProperties = (imageProps?: ImageProps) => {
+  const image =
+    imageProps?.path || imageProps?.locator
+      ? imageProps
+      : {
+          height: null,
+          width: 400,
+          altText: '',
+          path: '/cpsprodpb/36D1/production/_127933041__63970643_bbc-news-world-service-logo-nc.png',
+          locator: null,
+          originCode: null,
+          copyright: null,
+        };
```

Comment:
Is it worth storing this image in it's own variable so the intention is a little clear so: 
```javascript
const defaultImage = {
          height: null,
          width: 400,
          altText: '',
          path: '/cpsprodpb/36D1/production/_127933041__63970643_bbc-news-world-service-logo-nc.png',
          locator: null,
          originCode: null,
          copyright: null,
        };
```